gdnative-core/src/core_types/mod.rs

```diff
@@ -22,6 +25,9 @@ pub mod vector2;
 pub mod vector2_array;
```

Comment:
This is more a comment towards @toasteater and the dev team, not specific to this PR:

Do we export both the inner module and `core_types`? In my opinion, it could make sense to keep these `mod` declarations private, and just the `pub mod` re-export public.

Reply:
see also https://github.com/godot-rust/godot-rust/pull/713#discussion_r597551276